packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -107,6 +129,49 @@ void main() {
     expect(tester.testTextInput.setClientArgs!['inputAction'], equals(serializedActionName));
   }
 
+  // Related issue: https://github.com/flutter/flutter/issues/98115
+  testWidgets('ScheduleShowCaretOnScreen with no animation when the window changes metrics', (WidgetTester tester) async {
+    final ScrollController scrollController = ScrollController();
+    final Widget widget = MaterialApp(
+      home: Scaffold(
+        body: SingleChildScrollView(
+          controller: scrollController,
+          child: Column(
+            children: <Widget>[
+              Column(
+                children: List<Widget>.generate(
+                  5,
+                  (_) => Container(
```

Comment:
Sorry I didn't catch this earlier, our style guide suggests
`only use ⇒ when everything, including the function declaration, fits on a single line.`
can you convert this to a regular function?

Reply:
done